G-Chilie/Banka-UI
# #165482917 Build out Admin/Staff can View List of  Accounts Owned by a Specific Bank Accounts Endpoint (CRUD)

#### What does this PR do?

it can be Implements functionality where Admin/staff can view list of accounts owned by a specific user

#### Description of Task to be completed?

have this endpoint working
- GET /api/v1/user/john@gmail.com/accounts

#### How should this be manually tested?

- Clone https://github.com/G-Chilie/Banka.git
-  On your terminal cd into the banka folder
-  Run npm install
-  Run npm run before:start
-  Run npm dev
-  On postman make a POST request to _http://localhost:3000/api/v1/user/john@gmail.com/accounts
NB see screenshot below on the request data to pass

#### What are the relevant pivotal tracker stories?

[#165482917
](https://www.pivotaltracker.com/story/show/165482917)

#### Screenshots (if appropriate)

![get-user-account](https://user-images.githubusercontent.com/39604096/56635270-f2521200-665c-11e9-994a-f9c9c26caa5c.png)
